### PR TITLE
[Edge WS ] Principles

### PR DESCRIPTION
**Pull request by ASawwaf**
@CsatariGergely @TFredberg @trevgc 

Could you plz check

### CODE REVIEW

File: doc/tech/usecases.md [hunk 1]
```diff
@@ -11,3 +11,20 @@
 
 <a name="5.1"></a>
 ## 5.1 Edge
+4-Principle :
```
Comment (CsatariGergely) [inline doc/tech/usecases.md:14]: This should be a sub chapter of 5.1, like 5.1.1 or something.

File: doc/tech/usecases.md [hunk 2]
```diff
@@ -11,3 +11,20 @@
...
+
+This section introduce some principles that should be followed during developing and defining Edge scope for Reference Model , Reference Architecture 01 -OpenStack- , Reference Architecture 02 - K8s – down to Reference implementation and certificate under CNTT
+
+As we are not develop something new , so Edge Principle will follow Reference Model principle “ CNTT Reference Model:Introduction:Principles”  , Reference Architecture principle  “CNTT Reference Architecture : Principles” and Network principle  which in Nutshell
```
Comment (CsatariGergely) [inline doc/tech/usecases.md:18]: - nit: we are not developing
- Please add a link ot the referred chapters

File: doc/tech/usecases.md [hunk 3]
```diff
@@ -11,3 +11,20 @@
 
 <a name="5.1"></a>
 ## 5.1 Edge
+4-Principle :
+
+This section introduce some principles that should be followed during developing and defining Edge scope for Reference Model , Reference Architecture 01 -OpenStack- , Reference Architecture 02 - K8s – down to Reference implementation and certificate under CNTT
+
+As we are not develop something new , so Edge Principle will follow Reference Model principle “ CNTT Reference Model:Introduction:Principles”  , Reference Architecture principle  “CNTT Reference Architecture : Principles” and Network principle  which in Nutshell 
+
+-	Open source preference
```
Comment (CsatariGergely) [inline doc/tech/usecases.md:20]: Maybe some sentences would be good what is the meaning of these from edge point of view.

**Comment by ASawwaf** [comment]
@CsatariGergely @TFredberg @trevgc

any comments to merge

**Comment by markshostak** [comment]
@ASawwaf Hi Ahmed, Can we just inherit the principles from the RM, the way all of the other documents do? I'd suggest we limit any principles captured here to only ones that are **_unique_** to Edge. Otherwise, we're duplicating effort and creating a maintenance hassle. Thx, -Mark

**Comment by ASawwaf** [comment]
@markshostak , Make a lot of sense, that why we have a separate PR to consolidate all principal parts/section from RM/RA/NFG/Edge to be in one place 

Until having a consensus on the above approach, let move forward if you don't mind

**Comment by ASawwaf** [comment]
@CsatariGergely @TFredberg @trevgc

any further comments

File: doc/tech/usecases.md [hunk 4]
```diff
@@ -13,18 +13,32 @@
...
--	Security compliance:
+-  Geographical presence and data origin
+-  Data locality, protection and regulatory fulfilments
 -	Resilience and Availability
--	Scalability 
+-  WAN connectivity availability, capabilities and quality
+-  Autonomous local infrastructure functionality and operations
+-  Heterogeneous Infrastructure
 -	Workload diversity 
+-  Support of Telco and 3PP applications
```
Comment (pgoyal01) [inline doc/tech/usecases.md:43]: 3PP acronym

**Comment by ASawwaf** [comment]
Thanks @TFredberg

**Comment by rabi-abdel** [comment]
@ASawwaf any PR that needs merge needs to have the WIP label removed. please remove if you would like this PR to be approved/merged.

**Comment by ASawwaf** [comment]
Done

File: doc/tech/usecases.md (hunk 4 above)
Comment (ASawwaf) [reply doc/tech/usecases.md]: it is 3rd part application, but I tried to edit it but I didn't found it

Comment (ASawwaf) [reply doc/tech/usecases.md]: 3PP , 3rd party aplication

File: doc/tech/usecases.md [hunk 5]
```diff
@@ -9,5 +9,36 @@
...
+
+A main principle is that we shall not define a new branch of CNTT and refrain from inventing new things where other organizations already have solutions or ongoing suitable standardization. By that follows that the Edge Principles, as far as possible, shall follow and amend the existing [Reference Model Principles](../ref_model/chapters/chapter01.md#13-principles), the [Reference Architecture Principles](../ref_arch#principles) and the Network Principles that are being developed in parallel. 
+
+There are however a couple of new things that Edge deployments bring into the picture compared to the larger or regional Cloud Infrastructure deployments. These differences will generate a small set of additional principles. For now, there are only a few high level mixed differences and principles listed to generate suitable debate and feedback, but it will be filled out with more clarifications over time and during the development of the prioritized use cases.
```
Comment (pgoyal01) [inline doc/tech/usecases.md:20]: what does this mean, "For now, there are only a few high level mixed differences .." -- what are "mixed differences"?

File: doc/tech/usecases.md [hunk 6]
```diff
@@ -9,5 +9,36 @@
...
+
+-	Distribution into many small sites
```
Comment (pgoyal01) [inline doc/tech/usecases.md:22]: Many of the principles listed here are already included in, say, RA principles.

File: doc/tech/usecases.md (hunk 5 above)
Comment (ASawwaf) [reply doc/tech/usecases.md:20]: i don't know :):):):):)
it is completely not understandable :):)

let me try to rephrase it

File: doc/tech/usecases.md (hunk 6 above)
Comment (ASawwaf) [reply doc/tech/usecases.md:22]: yes in RA and RM , i stole it from thier, till we define a ONE principles covers all

**Comment by rabi-abdel** [comment]
PR moved to https://github.com/cntt-n/CNTT/pull/1449 to avoid confusion.

**Pull request closed without merging**